frappe/core/doctype/user/user.json

```diff
@@ -211,14 +211,6 @@
    "permlevel": 1,
    "read_only": 1
   },
-  {
-   "allow_in_quick_entry": 1,
-   "fieldname": "role_profile_name",
-   "fieldtype": "Link",
-   "label": "Role Profile",
-   "options": "Role Profile",
-   "permlevel": 1
-  },
```

Comment:
Let's not remove this right now. Better solution for backward compatibility:

1. Add new field.
2. On save old field is moved to new filed automatically (also done via patch)
3. Hide the old field and we remove it completely in v16. 


If we don't do this it will break all current code that programmatically assigns role profiles.

Reply:
> On save old field is moved to new filed automatically (also done via patch) 
This is done via patch.

Reply:
We should still do it on_save because existing code doesn't know anything about this table.